src/tests/Rockets.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import store from '../Redux/configureStore';
+import Rockets from '../components/Rockets';
+import Rocket from '../components/Rocket';
+import rocketsReducer, { showRockets, addRocketReservation, cancelRocketReservation } from '../redux/rockets/rockets';
+import rocketTestData from './testData';
```

Comment:
- [ ] Your tests fail because of the imports on lines 4 and 8. `Redux` at line 4 should be `redux` and `testdata` at line 8 should be `testdata`. Please fix that.

<img width="463" alt="image" src="https://user-images.githubusercontent.com/77190203/176909478-f2ac4811-7764-4d2e-b99b-c17482a9bc12.png">